src/rust/protocol/fuzz/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+
+[package]
+name = "protocol-fuzz"
+version = "0.0.0"
+authors = ["Automatically generated"]
+publish = false
+edition = "2018"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+libfuzzer-sys = "0.4"
+
+[dependencies.protocol]
+path = ".."
+
+# Prevent this from interfering with workspaces
+[workspace]
+members = ["."]
+
+[[bin]]
+name = "fuzz_target_1"
```

Comment:
What's this for?

Reply:
Need to remove that one. Was the default target that cargo fuzz init created

Reply:
Fixed